ssd1306/ssd1306.py

```diff
@@ -185,3 +185,26 @@ def contrast(self, contrast):
     self.write_command(SETCONTRAST)
     self.write_command(contrast)
 
+  def set_start_end_cols(self, start_col=0, end_col=None):
+    if end_col is None:
+      end_col = self.columns - 1
+    if start_col < 0 or start_col > self.columns - 1:
+      raise ValueError('Start column must be between 0 and %d.' % (self.columns - 1,))
+    if end_col < 0 or end_col > self.columns -1:
+      raise ValueError('End column must be between 0 and %d.' % (self.columns - 1,))
+
```

Comment:
The start/end location for pages and columns is an inclusive range. For example, if you call `set_start_end_cols(0, 0)` you'll write to only column zero. How this actually writes to the screen depends on the addressing mode (see pgs. 35-36 of the [datasheet](https://www.adafruit.com/datasheets/SSD1306.pdf)), but the bottom line is that it doesn't have to be a strict inequality.

But you're definitely right, and I appreciate you pointing that out. I'll add another check to make sure that `end_page` isn't less than `start_page` and same for the columns.